testing/zerotier-one/ppc64le-make-linux.patch

```diff
@@ -0,0 +1,14 @@
+--- a/make-linux.mk
++++ b/make-linux.mk
+@@ -154,6 +154,11 @@
+         ZT_ARCHITECTURE=6
+ 	override DEFS+=-DZT_NO_TYPE_PUNNING
+ endif
++ifeq ($(CC_MACH),powerpc64le)
++        ZT_ARCHITECTURE=7
++        override DEFS+=-DZT_NO_TYPE_PUNNING
++endif
```

Comment:
Is this reported and sent upstream?

Reply:
I've opened an issue:
https://github.com/zerotier/ZeroTierOne/issues/701